docs/node/hooks/jitter_calc.md

```diff
@@ -4,6 +4,8 @@ hide_table_of_contents: true
 
 # Calculate Jitter
 
+The `jitter_calc` hook is designed to calculate the jitter, mean, and variance between GPS and NTP timestamps.
```

Comment:
I dont think it has any notion of GPS and NTP timestamps..
Can you check the code? I believe its calculating jitter based on inter-sample timing?

Reply:
Yes, you are right. I changed the docu accordingly.